app/models/answer.rb

```diff
@@ -0,0 +1,7 @@
+class Answer < ActiveRecord::Base
+  belongs_to :user
+  belongs_to :survey
+  belongs_to :question
+  belongs_to :choice
+  validates :user_id, :survey_id, :question_id, :choice_id, presence: true
```

Comment:
You should probably have a uniqueness validation between user and question so that users (at the model level) can't answer the same question twice...

```ruby
validates_uniqueness_of :user, :scope => :question
# or
validates :uniqueness, :user, {scope: :question}
```